Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,312 @@
+
+        if issue_create_time > last_run_time:
+            incidents.append({
+                'name': str(issue['number']) + " - GitHub integration",
+                'occurred': issue['created_at'],
+                'rawJSON': json.dumps(issue)
+            })
+
+        if issue_create_time > last_run_time:
+            last_run_time = issue_create_time
+
+    demisto.incidents(incidents)
+    demisto.setLastRun({'time': last_run_time.isoformat().split('.')[0] + 'Z'})
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+def get_issue_context(issue):
+    label_context = []
+    for l in issue['labels']:
+        label_context.append(l['name'])
+
+    assignee_context = []
+    for a in issue['assignees']:
+        assignee_context.append(a['login'])
+
+    context = {
+        'ID': issue['number'],
+        'title': issue['title'],
+        'state': issue['state'],
+        'locked': issue['locked'],
+        'body': issue['body'],
+        'assignees': assignee_context,
+        'labels': label_context
+    }
+    return context
+
+
+def List_all_issues(show_all):
+    suffix = '/repos/{}/{}/issues'.format(OWNER, REPO)
+    if show_all == 'false':
+        res = http_request('GET', suffix, None, None)
+    else:
+        res = http_request('GET', suffix, None, {'state': 'all'})  # include close issues
+
+    return res
+
+
+def List_all_issues_command():
+    show_all = demisto.args()['show-all']
+    issues = List_all_issues(show_all)
+    issue_list = []
+    for s in issues:
+        context = get_issue_context(s)
+        issue_list.insert(0, context)
+
+    header_list = ['ID', 'title', 'state', 'locked', 'body', 'assignees', 'labels']
+    human_readable = tableToMarkdown('issues list: ', issue_list, header_list)
+    entry_context = {
+        'Git.issue(val.ID && val.ID == obj.ID)': issue_list
+    }
+
+    return_outputs(human_readable, entry_context, issues)
+
+
+def create_issue(args):
+    suffix = '/repos/{}/{}/issues'.format(OWNER, REPO)
+    data = {}
+    for key in args.keys():
+        data[key] = args[key]
+
+    if 'labels' in data:
+        data['labels'] = data['labels'].split(',')
+
+    if 'assignees' in data:
+        data['assignees'] = data['assignees'].split(',')
+
+    data = json.dumps(data)
+    # return_error(data)
+    res = http_request('POST', suffix, data, None)
+    return res
+
+
+def create_issue_command():
+    args = demisto.args()
+    res = create_issue(args)
+    context = get_issue_context(res)
+
+    header_list = ['ID', 'title', 'state', 'locked', 'body', 'assignees', 'labels']
+    human_readable = tableToMarkdown('created issue successfully: ', context, header_list, removeNull=True)
+    entry_context = {
+        'Git.issue(val.id && val.id == obj.id)': context
+    }
+
+    return_outputs(human_readable, entry_context, res)
+
+
+def close_issue(issue_id):
+    suffix = '/repos/{}/{}/issues/{}'.format(OWNER, REPO, issue_id)
```

Comment:
I think it would be nicer to have all the suffixes as CONST and then just to fill them in the functions